P5/Source/Guidelines/en/HD-Header.xml

```diff
@@ -1799,10 +1799,16 @@ more than one language, as in the following example:
       <specList>
         <specDesc key="unitDef"/>
         <specDesc key="unit"/>
-        <specDesc key="conversion" atts="formula fromUnit toUnit from to notBefore notAfter when"/> 
+        <specDesc key="conversion" atts="fromUnit toUnit from to notBefore notAfter when"/> 
         <specDesc key="att.formula" atts="formula"/>
+        <specDesc key="att.locatable" atts="where"/>
       </specList>
-Within the <gi>unitDef</gi>, a <gi>conversion</gi> element may be used to store information relating to conversion between units. The <gi>conversion</gi> element holds a special pair of attributes, <att>fromUnit</att> and <att>toUnit</att>, which serve to indicate the direction of a calculation from one unit of measure (stored in <att>fromUnit</att>) to another (stored in <att>toUnit</att>). A mathematical calculation to define the relation between these units may be stored in <att>formula</att>, as shown in the following examples. The <att>formula</att> attribute takes a value expressed as an XPath expression, which means that division must be expressed with <q>div</q> so as not to be confused with the forward slash used in path navigation. 
+Within the <gi>unitDef</gi>, a <gi>conversion</gi> element may be used to store information relating to conversion between units. 
+      The <gi>conversion</gi> element holds a special pair of attributes, <att>fromUnit</att> and <att>toUnit</att>, which serve to indicate the direction of a calculation from one unit of measure (stored in <att>fromUnit</att>) to another (stored in <att>toUnit</att>). 
+      A mathematical calculation to define the relation between these units may be stored in <att>formula</att>, as shown in the following examples.  
+      The <att>formula</att> attribute takes a value expressed as an XPath expression, for which see <ptr target="#XPATH31"/>. 
+      Among other things, this means that division must be expressed with <q>div</q> as opposed to the more common forward slash (<q>/</q>, U+002F), so as not to be confused with path navigation.
+      The <gi>conversion</gi> element may also use attributes to store dates and date ranges as well as location(s) that specify when and where a conversion formula was applied.
```

Comment:
@sydb , really, this statement (line 1811) covers *exactly* what we mean. I don't really think it requires anything more to clarify.